app/components/edit/EditNote.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+
+/* eslint-disable react/no-multi-comp */
+class EditNote extends Component {
```

Comment:
I extracted before disabling the `max-classes-per-file` rule. Other files that violated the rule had components that were more tightly coupled. That's worth refactoring imo, but out of scope here (also imo). I ended up opting to disable the rule, but left this refactor since it was already done.